docs/index.rst

```diff
@@ -19,11 +19,19 @@ NLP. Otherwise, please refer to an introduction course such as
 
 .. note::
 
-   This project is at an early stage and is under active development.
-   Please expect that it will be updated frequently. Contributions are welcome.
+   We can be found on `Github <https://github.com/dmlc/gluon-nlp>`_. This project is at an early stage
```

Comment:
We should show the list of papers/models we implemented (e.g. https://github.com/pytorch/fairseq#introduction)